pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -40,6 +42,7 @@ const (
 	kubeProxyPath = k8sDir + "kube-proxy.exe"
 	// hybridOverlayPath is the location of the hybrid-overlay-node exe
 	hybridOverlayPath = k8sDir + "hybrid-overlay-node.exe"
+
```

Comment:
extra line

Reply:
I put the extra line there to seperate paths and variables

Reply:
Please add comment then otherwise folks are going to mix this up in the future.